frontend/packages/pipelines-plugin/src/utils/pipeline-filter-reducer.ts

```diff
@@ -61,5 +61,11 @@ export const pipelineResourceTypeFilter = (filters, pipelineResource): boolean =
 
 export const taskRunFilterReducer = (taskRun): string => {
   const status = pipelineRunStatus(taskRun);
+  if (status === 'Running') {
+    const condition = taskRun.status?.conditions?.find((c) => c.type === 'Succeeded');
+    if (condition.reason === 'ExceededNodeResources') {
+      return 'Pending';
+    }
+  }
   return status || '-';
```

Comment:
That's not part of your change, but thats exactly the reason I would prefer to keep TaskRun and PipelineRuns separated. But for now this change is "technical fine".

But there are more reasons why a TaskRun is Pending... (see https://github.com/tektoncd/pipeline/blob/66096db2c12125693c39b32bb1cbfeff6cfdb644/pkg/pod/status.go#L284-L303).

So we need to check the other both states here as well (`CreateContainerConfigError` and `Pending`) or check for status == `Unknown` and reason != `Pending`. Maybe its safer to define an array of "taskRunPendingReasons"? Wdyt?